soccer/main.py

```diff
@@ -14,6 +14,10 @@
 LEAGUE_IDS = leagueids.LEAGUE_IDS
 TEAM_NAMES = teamnames.team_names
 
+session = requests.Session()
```

Comment:
Because Betamax needs to piggy back into the same request session that the application uses. Depending on the configuration, the first time it detects a new request, it will perform it and store the results into a file (called cassettes). Then the second time that request needs to be performed, it will monkeypatch `request` so that instead of hitting the server, it will read from the previously created file.